drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -848,32 +850,32 @@ static struct ieee802154_radio_api rf2xx_radio_api = {
 	static const struct rf2xx_config rf2xx_ctx_config_##n = {		   \
 		.inst = n,							   \
 										   \
-		.irq.devname = DT_INST_##n##_ATMEL_RF2XX_IRQ_GPIOS_CONTROLLER,	   \
-		.irq.pin = DT_INST_##n##_ATMEL_RF2XX_IRQ_GPIOS_PIN,		   \
-		.irq.flags = DT_INST_##n##_ATMEL_RF2XX_IRQ_GPIOS_FLAGS,		   \
+		.irq.devname = DT_INST_GPIO_LABEL(n, irq_gpios),	   \
+		.irq.pin = DT_INST_GPIO_PIN(n, irq_gpios),		   \
+		.irq.flags = DT_INST_GPIO_FLAGS(n, irq_gpios),		   \
```

Comment:
Only cosmetics...please keep *\\* align on all changes.

Reply:
be aware the conversion is being done by a script, so thus the formatting for such things isn't maintained.  I'll fix this one up by hand.